terraform/modules/networking/variables.tf

```diff
@@ -18,11 +18,39 @@ variable "location" {
   type        = string
 }
 
-variable "subnets" {
-  description = "List of subnets to create"
+variable "virtual_machines" {
+  description = "List of virtual machines with their configurations"
   type = list(object({
-    name   = string
-    prefix = string
+    name          = string
+    database_name = string
+    disk_size     = number
+    networking = object({
```

Comment:
Vm-urile si networking-ul sunt lucuri diferite, nu isi au locul in acelasi map. Un map the vm-uri, un map de networking in care sa fie subnets cu network security si poate network interface (pentru cazul nostru nu e nevoie sa avem map de vnets pentru ca folosim doart unul), un map de ip-uri
Ne complicam prea tare asa si este foarte greu de citit ce se intampla